src/startExpressServer.ts

```diff
@@ -100,14 +100,17 @@ export const startExpressServer = (
         event = rest.interactive.type;
         data = {
           ...(rest.interactive.list_reply || rest.interactive.button_reply),
-          context: rest.context,
         };
         break;
 
       default:
         break;
     }
 
+    if (rest.context) {
+      data!.context ??= rest.context;
```

Comment:
Can we use an immutable operation?

```js
if (rest.context) {
  data = {
    ...data,
    context: rest.context,
  }
}
```

Reply:
Sure!

Reply:
Awesome. Happy to approve once you do the change :)